js/templates/settings.html

```diff
@@ -126,6 +126,27 @@ <h3 class="padding15 margin0 fontWeight500"><%= polyglot.t('General') %></h3>
             <div class="flexRow">
               <div class="flexCol-3  borderRight custCol-border">
                 <div class="fieldItem">
+                  <label><%= polyglot.t('BitcoinUnit') %></label>
+                </div>
+              </div>
+
+              <div class="flexCol-9 borderRight0 custCol-border">
+                <div class="fieldItem">
+                  <input type="radio" class="fieldItem" id="bitcoin_unit_BTC" name="bitcoinUnit" value="BTC" />
```

Comment:
Can we please use camel-case for the id's (and consequently the 'for' attributes)? Camel-case is our convention in 95% of cases - the only exception may be in the model layer where the server is giving us the underscored format.